docs/rules/no-force-true.md

```diff
@@ -0,0 +1,37 @@
+# disallow using of &#39;force: true&#39; option (no-force-true)
+
+Using force:true on inputs appears to be confusing rather than helpful.
+It usually silence the actual problem instead of providing a way to overcome it.
+See [Cypress Core Concepts](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Forcing).
+
+
+## Rule Details
+
+This rule aims to disallow using of force option on inputs.
+
+Examples of **incorrect** code for this rule:
+
+```js
+
+cy.get('button').click({force: true})
+cy.get('input').type('somth', {force: true})
+
+```
+
+Examples of **correct** code for this rule:
+
+```js
+
+cy.get('button').click()
+cy.get('button').click({multiple: true})
+
+```
+
+
+## When Not To Use It
+
+If you really need to use.
+
+## Further Reading
+
+If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.
```

Comment:
Remove this last line section, it is placeholder text

Reply:
done